### PR TITLE
Curated feeds

### PR DESCRIPTION
**Pull request by eamonma**
Add curated feeds section in AddFeedActivity

### CODE REVIEW

**Review by ismail-ahmed0149 — APPROVED** [review]
I really liked the idea of having curated feeds. Everything seems to be looking good and the code is well-formatted. Great work!

File: app/src/main/java/monster/minions/binocularss/ui/CuratedFeeds.kt [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+val feeds = listOf(
+    Feed(source = "https://rss.cbc.ca/lineup/topstories.xml", title = "CBC Top Stories"),
+    Feed(source = "https://rss.cbc.ca/lineup/world.xml", title = "CBC World"),
+    Feed(source = "https://rss.cbc.ca/lineup/canada.xml", title = "CBC Canada"),
+    Feed(source = "https://rss.cbc.ca/lineup/politics.xml", title = "CBC Politics"),
+    Feed(source = "https://androidauthority.com/feed", title = "Android Authority"),
+    Feed(
+        source = "https://www.ctvnews.ca/rss/ctvnews-ca-top-stories-public-rss-1.822009",
+        title = "CTV Top Stories"
+    ),
```
Comment (hisbaan) [inline app/src/main/java/monster/minions/binocularss/ui/CuratedFeeds.kt:31]: I would suggest moving this into AddFeedActivity and passing it as an argument that way the composable file remains purely a composable file (single responsibility principle and all that)

**Review by hisbaan — APPROVED** [review]
Good work! I've tested it out and it looks very clean. I really like how they no longer remain in the list once you have added them, that is a very good UX choice. Once you address the comment that I added, you're good to merge in my eyes. Awesome job!

File: app/src/main/java/monster/minions/binocularss/ui/CuratedFeeds.kt (hunk 1 above)
Comment (eamonma) [reply app/src/main/java/monster/minions/binocularss/ui/CuratedFeeds.kt]: Done

**Pull request merged**